### ISSUE TITLE
[Sourcing][Accessibility][ATL testing][Multi-input mobile mode] This multi-input ‘Field placeholder’ and ‘Close’ button, is not reachable with backward navigation using shift+ tab key once after navigating within the items using arrow keys.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10384]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
This multi-input ‘Field placeholder’ and ‘Close’ button, is not reachable with backward navigation using shift+ tab key once after navigating within the items using arrow keys.
<img width="451" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/88182358/41e99520-bc5b-4d2d-bb83-df96525054b6">

#### If this is a bug, please provide steps for reproducing it.
tested on https://sap.github.io/fundamental-ngx/#/core/multi-input#mobile
#### Is there anything else we should know?
Reported by ATL

**Comment by droshev** [comment]
I cannot reproduce it

![Image](https://github.com/SAP/fundamental-ngx/assets/4380815/72c18e3e-320e-4c99-ba14-518aaf7ac051)

**Comment by I543348** [comment]
Hi @droshev , by backward navigation - we are talking about accessibility using JAWS.
When we turn on JAWS and press tab + shift -> the focus is not going back to the previous elements.
This is reported by ATL testing.

**Comment by droshev** [comment]
@I543348 The mobile example is not created to be tested with JAWS

**Comment by I543348** [comment]
ATL Re-verification needed

**Comment by sagar7259** [comment]
Hello,

Now on backward navigation, the input ‘Field placeholder’ is reachable using keyboard, but the 'Close' button at the top is still not reachable using keyboard.

@rvamudha please reopen this ticket.

Regards,
Sagar (ATL team)

**Comment by droshev** [comment]
@I543348 The mobile example is not created to be tested with JAWS

**Comment by sagar7259** [comment]
Hi Team,

The reported issue is now fixed has the 'Close' button is under tab chain. Hence this ticket can be closed.

Regards,
Sagar
